packages/core/src/helpers.ts

```diff
@@ -220,7 +220,7 @@ export default class Helpers implements Resolver {
      * Helper function to remove reserved keywords in required property of TD JSON Schema
      */
     static createExposeThingInitSchema(tdSchema: unknown): SomeJSONSchema {
-        const tdSchemaCopy = JSON.parse(JSON.stringify(tdSchema));
+        const tdSchemaCopy = structuredClone(tdSchema) as SomeJSONSchema;
```

Comment:
I added this as another instance suitable for using the `structuredClone` function – the typing is not that nice here, but I think casting should be fine here since we are performing type checks down below.

Reply:
I guess it is acceptable, we could even put it as the argument type given the fact that is just an internal helper function